spec/domain/accounts/tasks/notify_spec.exs

```diff
@@ -0,0 +1,135 @@
+defmodule Squadster.Accounts.Tasks.NotifySpec do
+  use ESpec.Phoenix, async: true
+  use ESpec.Phoenix.Extend, :domain
+
+  import Mockery
+  import Mockery.Assertions
+
+  alias Squadster.Accounts.Tasks.Notify
+
+  let :squad, do: insert(:squad)
+
+  describe "notify/1" do
+    let :message, do: "Who is John Galt?"
+    let :user, do: insert(:user, id: 1)
+
+    before do
+      mock HTTPoison, :post
+    end
+
+    context "when the target is user" do
+      it "sends message to this user" do
+        Notify.notify(message: message(), target: user())
+        assert_called HTTPoison, :post, [
+          _endpoint,
+          """
+            {
+              "text": "Who is John Galt?",
+              "target": 1
```

Comment:
It doesn't allow matching with function calls inside, only macros, but due to some elixir magic I could not set macro for dynamic id injection, so let it be hard-coded for now